src/platform/windows/events_loop.rs

```diff
@@ -259,12 +283,21 @@ thread_local!(static CONTEXT_STASH: RefCell<Option<ThreadLocalData>> = RefCell::
 struct ThreadLocalData {
     sender: mpsc::Sender<Event>,
     windows: HashMap<winapi::HWND, Arc<Mutex<WindowState>>>,
+    win32_block_loop: Arc<(Mutex<bool>, Condvar)>
+}
+
+fn event_is_resize(event: &Event) -> bool {
```

Comment:
I can move the check out of a function, but it can't happen as an `if` clause because `Event` doesn't implement `Copy` and is consumed by `callback(event)`.